src/test_l3_dsHdmiIn.c

```diff
@@ -0,0 +1,1396 @@
+ */
+static void hdmiInAllmChangeCB(dsHdmiInPort_t port, bool allm_mode )
+{
+    UT_LOG_INFO("Received AllmChange status callback port: %s, sigstatus: %s",
+                 UT_Control_GetMapString(dsHdmiInPort_mapTable, port),
+                 UT_Control_GetMapString(bool_mapTable, allm_mode));
+
+    gAllmmode = allm_mode;
+}
+
+/**
+ * @brief Callback function for HdmiIn Allmmode change.
+ *
+ * This function is invoked whenever a Allmmode change occurs in the Hdmi connection.
+ */
+static void hdmiInAVLatencyChangeCB(int audio_latency, int video_latency)
+{
+    UT_LOG_INFO("Received AVlatencyChange status callback audio_latency: %d\t"
+                 "video_latency:%d \t", audio_latency , video_latency);
+
+    gaudio_latency = audio_latency;
+    gvideo_latency = video_latency;
+}
+
+/**
+ * @brief Callback function for HdmiIn Allmmode change.
+ *
+ * This function is invoked whenever a Allmmode change occurs in the Hdmi connection.
+ */
+static void hdmiInAviContentTypeChangeCB(dsHdmiInPort_t port, dsAviContentType_t avi_content_type)
+{
+    UT_LOG_INFO("Received AviContentType change callback port: %s, avi_content_type: %d\t"
+                 "video_latency:%d\t",
+                 UT_Control_GetMapString(dsHdmiInPort_mapTable, port),
+                 UT_Control_GetMapString(dsAviContentType_mapTable, avi_content_type)),
+
+    gavi_content_type = avi_content_type;
+}
+
+
+/**
+* @brief This test initializes the HdmiIn Module.
+*
+* This test function initializes the HdmiIn Module.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 001@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_initialize(void)
+{
+    gTestID = 1;
+    UT_LOG_INFO("In %s [%02d%03d]", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret = dsERR_NONE;
+
+    /* Initialize the HdmiIn Module */
+    UT_LOG_INFO("Calling dsHdmiInInit()");
+    ret = dsHdmiInInit();
+    UT_LOG_INFO("Result dsHdmiInInit() dsError_t:[%s]", UT_Control_GetMapString(dsError_mapTable, ret));
+    DS_ASSERT(ret == dsERR_NONE);
+
+    /* Register connection status callback */
+    UT_LOG_INFO("Calling dsHdmiInRegisterConnectCB(IN:CBFunc:[0x%0X])", hdmiInConnectCB);
+    ret = dsHdmiInRegisterConnectCB(hdmiInConnectCB);
+    UT_LOG_INFO("Result dsHdmiInRegisterConnectCB(IN:CBFunc:[0x%0X]) dsError_t:[%s]",
+                        hdmiInConnectCB, UT_Control_GetMapString(dsError_mapTable, ret));
+    DS_ASSERT(ret == dsERR_NONE);
+
+    /* Register Signal change callback */
+    if(gSourceType == 0)
+    {
+        UT_LOG_INFO("Calling dsHdmiInRegisterSignalChangeC(IN:CBFun:[0x%0X])", hdmiInSignalChangeCB);
+        ret = dsHdmiInRegisterSignalChangeCB(hdmiInSignalChangeCB);
+        UT_LOG_INFO("Result dsHdmiInRegisterSignalChangeC(IN:CBFun:[0x%0X]) dsError_t:[%s]",
+                            hdmiInSignalChangeCB, UT_Control_GetMapString(dsError_mapTable, ret));
```

Comment:
typo mistake  **dsHdmiInRegisterSignalChangeC(** in log print.

Reply:
done